docs-requirements.txt

```diff
@@ -1,3 +1,5 @@
 jsonschema2md==0.4.0
+fastjsonschema==2.16.2
```

Comment:
Why do you need these in docs?

Reply:
Docs use the `json_schema.py` file to generate the docs, I moved the compilation of the schemas into validators into that file. All of that files dependencies need to be installed to run the docs command, I could move it out to another new file if this isn't a good solution.